diff --git a/node_modules/@psychedelic/dab-js/dist/registries/nfts_registry.js b/node_modules/@psychedelic/dab-js/dist/registries/nfts_registry.js
index a86e016..60bb9db 100644
--- a/node_modules/@psychedelic/dab-js/dist/registries/nfts_registry.js
+++ b/node_modules/@psychedelic/dab-js/dist/registries/nfts_registry.js
@@ -76,7 +76,7 @@ const getUserCollectionTokens = (collection, user, agent = DEFAULT_AGENT, callba
         return collectionDetails;
     }
     catch (e) {
-        console.error(e);
+        // console.error(e);
         return {
             name: collection.name,
             canisterId: collection.principal_id.toString(),
@@ -88,7 +88,7 @@ const getUserCollectionTokens = (collection, user, agent = DEFAULT_AGENT, callba
 exports.getUserCollectionTokens = getUserCollectionTokens;
 const getNFTActor = ({ canisterId, agent, standard }) => {
     if (!(standard in NFT_STANDARDS)) {
-        console.error(`Standard ${standard} is not implemented`);
+        // console.error(`Standard ${standard} is not implemented`);
         throw new Error(`standard is not supported: ${standard}`);
     }
     return new NFT_STANDARDS[standard](canisterId, agent);
